go-fmt-pwd

```diff
@@ -166,7 +166,7 @@ bin/syz-sysgen:
 format: format_go format_cpp format_sys
 
 format_go:
-	$(GO) fmt ./...
+	cd $(realpath ${PWD}) && $(GO) fmt ./...
```

Comment:
Please add a comment here about why we are doing this.

Reply:
So everything else works with this? In particular 'go build'?

Reply:
It's not clear to me where other issues may arise.

Yes, e.g. `make manager` (which invokes `go build`) does work fine, even without this change.